modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -382,6 +382,114 @@ definition {
 		}
 	}
 
+	@description = "LPS-165849 - Verify terms from a related custom object (child) are replaced correctly in an email notification"
```

Comment:
Verify terms from a child object are populated correctly in an email notification